.gitignore

```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 # build data
 public/data
 src/data.json
+
+# quickstart generate data shell file
```

Comment:
Why did you add this to gitignore? You're confusing me 😃

Reply:
it was marked as `modified` as soon as I forked/cloned; I've only seen that happen on files that constantly have non-breaking changes and should be ignored. I can revoke that if you'd like.

Reply:
You can remove this. For some reason the permissions got changed on the file from 755 to 644.

Reply:
reverted with commit bc609b6